addons/website_livechat/tests/test_chatbot_ui.py

```diff
@@ -25,6 +25,39 @@ def setUp(self):
 
         self.env.ref('website.default_website').channel_id = self.livechat_channel.id
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.chatbot_redirect_script = cls.env["chatbot.script"].create(
+            {"title": "Redirection Bot"}
+        )
+        question_step, _ = tuple(
+            cls.env["chatbot.script.step"].create([
+                {
+                    "chatbot_script_id": cls.chatbot_redirect_script.id,
+                    "message": "Hello, were do you want to go?",
+                    "step_type": "question_selection",
+                },
+                {
+                    "chatbot_script_id": cls.chatbot_redirect_script.id,
+                    "message": "Tadam, we are on the page you asked for!",
+                    "step_type": "text",
+                }
+            ])
+        )
+        cls.env["chatbot.script.answer"].create([
+            {
+                "name": "Go to the #chatbot-redirect anchor",
+                "redirect_link": "#chatbot-redirect",
+                "script_step_id": question_step.id,
+            },
+            {
+                "name": "Go to the /chabtot-redirect page",
+                "redirect_link": "/chatbot-redirect",
+                "script_step_id": question_step.id,
+            },
+        ])
```

Comment:
Sharing data between all tests significantly reduces test maintainability: harder to read test (need to see setup first, which is hidden), any change can undesirably impact other tests, etc.